src/profiles_management/helpers/kfam.py

```diff
@@ -0,0 +1,139 @@
+"""Utility module for manipulating KFAM resources."""
+
+import logging
+from typing import List
+
+from lightkube.core.exceptions import ApiError
+from lightkube.generic_resource import GenericNamespacedResource, create_namespaced_resource
+from lightkube.resources.rbac_authorization_v1 import RoleBinding
+
+from profiles_management.helpers.k8s import client, get_name, get_namespace
+
+log = logging.getLogger(__name__)
+
+AuthorizationPolicy = create_namespaced_resource(
+    group="security.istio.io",
+    version="v1beta1",
+    kind="AuthorizationPolicy",
+    plural="authorizationpolicies",
+)
+
+
+def list_contributor_rolebindings(namespace="") -> List[RoleBinding]:
+    """Return a list of KFAM RoleBindings.
+
+    Only RoleBindings which have "role" and "user" annotations will be returned.
+    The RoleBinding for the Profile owner, with name namespaceAdmin, will not be
+    returned."
+
+    Args:
+        namespace: The namespace to list contributors from. For all namespaces
+                   you can use "" value.
+
+    Returns:
+        A list of RoleBindings that are used from KFAM for contributors.
+    """
+    role_bindings = client.list(RoleBinding, namespace=namespace)
+    contributor_rbs = []
+    for rb in role_bindings:
+        if not rb.metadata:
+            continue
+
+        # We exclude the RB created by the Profile Controller for the
+        # owner of the Profile
+        # https://github.com/kubeflow/kubeflow/issues/6576
+        if rb.metadata.name == "namespaceAdmin":
+            continue
+
+        if not rb.metadata.annotations:
+            continue
+
+        if "role" not in rb.metadata.annotations:
+            continue
+
+        if "user" not in rb.metadata.annotations:
+            continue
+
+        contributor_rbs.append(rb)
+
+    return contributor_rbs
+
+
+def delete_contributor_rolebinding(rb: RoleBinding):
```

Comment:
After using `delete_many`, are these still needed?

Reply:
Maybe some browser cache issue?

I've removed the files alongside the `delete_many` commit in https://github.com/canonical/github-profiles-automator/commit/b90169923cf728ea946d39f8f31fd29ca5444641